src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -23,23 +24,23 @@
   (let [path (get-derivation-path number-of-accounts)]
     (format-derivation-path path)))
 
-(defn- calculate-raw-balance
-  [raw-balance decimals]
-  (if-let [n (utils.number/parse-int raw-balance nil)]
-    (/ n (Math/pow 10 (utils.number/parse-int decimals)))
-    0))
```

Comment:
`utils.money/token->unit` is able to do the same and works with big numbers, so now it is being used